tools/deployment/package/docker-compose.base.yaml

```diff
@@ -0,0 +1,361 @@
+    ]
+
+  compression-scheduler:
+    <<: *service_defaults
+    hostname: "compression_scheduler"
+    stop_grace_period: "300s"
+    environment:
+      BROKER_URL: "amqp://${CLP_QUEUE_USER:?Please set a value.}:\
+        ${CLP_QUEUE_PASS:?Please set a value.}@queue:5672"
+      CLP_DB_PASS: "${CLP_DB_PASS:?Please set a value.}"
+      CLP_DB_USER: "${CLP_DB_USER:?Please set a value.}"
+      CLP_LOGGING_LEVEL: "${CLP_COMPRESSION_SCHEDULER_LOGGING_LEVEL:-INFO}"
+      CLP_LOGS_DIR: "/var/log"
+      PYTHONPATH: "/opt/clp/lib/python3/site-packages"
+      RESULT_BACKEND: "redis://default:${CLP_REDIS_PASS:?Please set a value.}@redis:6379\
+        /${CLP_REDIS_BACKEND_DB_COMPRESSION:-1}"
+    volumes:
+      - *volume_clp_config_readonly
+      - *volume_clp_logs
+      - "${CLP_AWS_CONFIG_DIR_HOST:-empty}:/.aws:ro"
+      - "${CLP_LOGS_INPUT_DIR_HOST:-empty}:${CLP_LOGS_INPUT_DIR_CONTAINER:-/mnt/logs}"
+    depends_on:
+      db-table-creator:
+        condition: "service_completed_successfully"
+      queue:
+        condition: "service_healthy"
+      redis:
+        condition: "service_healthy"
+    command: [
+      "python3",
+      "-u",
+      "-m", "job_orchestration.scheduler.compress.compression_scheduler",
+      "--config", "/etc/clp-config.yml"
+    ]
+
+  compression-worker:
+    <<: *service_defaults
+    hostname: "compression_worker"
+    stop_grace_period: "60s"
+    environment:
+      BROKER_URL: "amqp://${CLP_QUEUE_USER:?Please set a value.}\
+        :${CLP_QUEUE_PASS:?Please set a value.}@queue:5672"
+      CLP_CONFIG_PATH: "/etc/clp-config.yml"
+      CLP_HOME: "/opt/clp"
+      CLP_LOGGING_LEVEL: "${CLP_COMPRESSION_WORKER_LOGGING_LEVEL:-INFO}"
+      CLP_LOGS_DIR: "/var/log/compression_worker"
+      CLP_WORKER_LOG_PATH: "/var/log/compression_worker/worker.log"
+      PYTHONPATH: "/opt/clp/lib/python3/site-packages"
+      RESULT_BACKEND: "redis://default:${CLP_REDIS_PASS:?Please set a value.}@redis:6379\
+        /${CLP_REDIS_BACKEND_DB_COMPRESSION:-1}"
+    volumes:
+      - *volume_clp_config_readonly
+      - *volume_clp_logs
+      - "${CLP_AWS_CONFIG_DIR_HOST:-empty}:/.aws:ro"
+      - "${CLP_LOGS_INPUT_DIR_HOST:-empty}:${CLP_LOGS_INPUT_DIR_CONTAINER:-/mnt/logs}"
+      - "${CLP_ARCHIVE_OUTPUT_DIR_HOST:-empty}\
+        :${CLP_STAGED_ARCHIVE_OUTPUT_DIR_HOST:+/tmp}/var/data/archives"
+      - "${CLP_STAGED_ARCHIVE_OUTPUT_DIR_HOST:-empty}\
+        :${CLP_ARCHIVE_OUTPUT_DIR_HOST:+/tmp}/var/data/staged-archives"
+      - type: "bind"
+        source: "${CLP_DATA_DIR_HOST:-./var/data}"
+        target: "/var/data"
+    command: [
+      "python3",
+      "-u",
+      "/opt/clp/lib/python3/site-packages/bin/celery",
+      "-A", "job_orchestration.executor.compress",
+      "worker",
+      "--concurrency", "${CLP_COMPRESSION_WORKER_CONCURRENCY:-1}",
+      "--loglevel", "WARNING",
+      "-f", "/var/log/compression_worker/worker.log",
+      "-Q", "compression",
+      "-n", "compression-worker"
+    ]
+
```

Comment:
the `/tmp` addition is to avoid docker creating the directory on the host with root ownership due to the `/var/data` mount below. not sure if it's a bug in docker or something